charts/newrelic-infrastructure/templates/daemonset-kubelet.yaml

```diff
@@ -0,0 +1,294 @@
+            {{- else }}
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "spec.nodeName"
+            {{- end }}
+            {{- end }}
+            {{- with .Values.customAttributes }}
+            - name: "NRIA_CUSTOM_ATTRIBUTES"
+              value: {{ . }}
+            {{- end }}
+            {{- if .Values.verboseLog }}
+            - name: NRIA_VERBOSE
+              value: "1"
+            {{- end }}
+            {{- with .Values.logFile }}
+            - name: NRIA_LOG_FILE
+              value: {{ . }}
+            {{- end }}
+            {{- if not .Values.privileged }}
+            - name: "NRIA_OVERRIDE_HOST_ROOT"
+              value: ""
+            - name: "NRIA_IS_SECURE_FORWARD_ONLY"
+              value: "true"
+            {{- end }}
+            {{- with .Values.kubelet.extraEnv }}
+            {{- tpl . $ | nindent 12 }}
+            {{- end }}
+          {{- with .Values.kubelet.extraEnvFrom }}
+          envFrom:
+            {{- tpl . $ | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            {{- if .Values.config }}
+            - name: config
+              mountPath: /etc/newrelic-infra.yml
+              subPath: newrelic-infra.yml
+            {{- end }}
+            {{- if .Values.integrations_config }}
+            - name: nri-integrations-cfg-volume
+              mountPath: /etc/newrelic-infra/integrations.d/
+            {{- end }}
+            {{- if .Values.privileged }}
+            - name: dev
+              mountPath: /dev
+            {{- if and (.Values.openshift.enabled) (eq .Values.openshift.version "4.x") }}
+            - name: host-crio-socket
+              mountPath: /var/run/crio.sock
+            {{- else }}
+            - name: host-docker-socket
+              mountPath: /var/run/docker.sock
+            {{- end }}
+            - name: log
+              mountPath: /var/log
+            - name: host-volume
+              mountPath: /host
+              readOnly: true
+            {{- else }}
+            - mountPath: /var/db/newrelic-infra/data
+              name: tmpfs-data
+            - mountPath: /var/db/newrelic-infra/user_data
+              name: tmpfs-user-data
+            - mountPath: /tmp
+              name: tmpfs-tmp
+            {{- end }}
+            {{- with .Values.kubelet.extraVolumeMounts }}
+            {{- tpl . $ | nindent 12 }}
+            {{- end }}
+          {{- with .Values.resources }}
+          resources: {{ toYaml . | nindent 12 }}
+          {{- end }}
+      volumes:
+        {{- if .Values.privileged }}
+        - name: dev
+          hostPath:
+            path: /dev
+        {{- if and (.Values.openshift.enabled) (eq .Values.openshift.version "4.x") }}
+        - name: host-crio-socket
+          hostPath:
+            path: /var/run/crio.sock
+        {{- else }}
+        - name: host-docker-socket
+          hostPath:
+            path: /var/run/docker.sock
```

Comment:
As we discussed, I'd love to be able to fully disable this in unprivileged and tell the agent to not log an error. Something to check with CAOS I think.